stetho/src/main/java/com/facebook/stetho/inspector/database/DatabasePeerManager.java

```diff
@@ -130,17 +168,19 @@ private SQLiteDatabase openDatabase(String databaseName) throws SQLiteException
 
   public interface ExecuteResultHandler<T> {
     public T handleResult(Cursor result) throws SQLiteException;
+
+    public T handleResult(Long value, String name) throws SQLiteException;
```

Comment:
Rename and clarify:
```java
handleSelect(Cursor result);
handleInsert(long lastInsertedId);
handleUpdateDelete(int rowsAffected);
```

The mapping to the DevTools response and is the job of the implementer of this interface.